tvision/rhide.env

```diff
@@ -2,7 +2,7 @@
 #   Common Warning and debug options, change it if you don't want debug
 # information or your compiler emmits too much warnings with -Wall.
 #
-RHIDE_OS_BOTH_Linux=-Wall -Werror -ggdb3 -O2
```

Comment:
-Werror says to treat all warnings as errors.  We don't want this, since this build currently encounters a warning.

Reply:
Which warning? Perhaps it can be fixed.